tgui/packages/tgui/interfaces/Pandemic.tsx

```diff
@@ -0,0 +1,484 @@
+                  <LabeledList.Item
+                    color={GetColor(virus.transmission)}
+                    label="Transmissibility">
+                    {virus.transmission}
+                  </LabeledList.Item>
+                </Tooltip>
+              </LabeledList>
+            </Section>
+          </Stack.Item>
+        </>
+      )}
+    </Stack>
+  );
+};
+
+/** Similar to the virus info display.
+ * Returns info about symptoms as collapsibles.
+ */
+const SymptomInfoDisplay = (props: SymptomDisplayProps) => {
+  const { symptoms } = props;
+
+  return !symptoms.length ? (
+    <NoticeBox>No symptoms detected.</NoticeBox>
+  ) : (
+    <Section fill title="Symptoms">
+      {symptoms.map((symptom) => {
+        return (
+          <Collapsible key={symptom.name} title={symptom.name}>
+            <Stack fill>
+              <Stack.Item grow={3}>
+                {symptom.desc}
+                <ThresholdDisplay thresholds={symptom.threshold_desc} />
+              </Stack.Item>
+              <Stack.Divider />
+              <Stack.Item grow={1}>
+                <Section title="Modifiers">
+                  <LabeledList>
+                    <Tooltip content="Rarity of the symptom.">
+                      <LabeledList.Item
+                        color={GetColor(symptom.level)}
+                        label="Level">
+                        {symptom.level}
+                      </LabeledList.Item>
+                    </Tooltip>
+                    <Tooltip content="Decides the cure complexity.">
+                      <LabeledList.Item
+                        color={GetColor(symptom.resistance)}
+                        label="Resistance">
+                        {symptom.resistance}
+                      </LabeledList.Item>
+                    </Tooltip>
+                    <Tooltip content="Symptomic progression.">
+                      <LabeledList.Item
+                        color={GetColor(symptom.stage_speed)}
+                        label="Stage Speed">
+                        {symptom.stage_speed}
+                      </LabeledList.Item>
+                    </Tooltip>
+                    <Tooltip content="Detection difficulty from medical equipment.">
+                      <LabeledList.Item
+                        color={GetColor(symptom.stealth)}
+                        label="Stealth">
+                        {symptom.stealth}
+                      </LabeledList.Item>
+                    </Tooltip>
+                    <Tooltip content="Decides the spread type.">
+                      <LabeledList.Item
+                        color={GetColor(symptom.transmission)}
+                        label="Transmission">
+                        {symptom.transmission}
+                      </LabeledList.Item>
+                    </Tooltip>
+                  </LabeledList>
+                </Section>
+              </Stack.Item>
+            </Stack>
+          </Collapsible>
```

Comment:
Again I feel like part of this can be compentised harder. Things are starting to get a little right-heavy again. The modifiers section can definitely be pulled into a component.

Reply:
if the tooltip wasn't hard coded i could also just iterate over the symptom/virus objects and make this code much simpler. maybe i'll do defines before hand, whatever, but i'm too fond of tooltip info to outright remove them

Reply:
Don't mind me thinking out loud, this is done